src/development/tools/sdk/release-notes/supported-platforms.md

```diff
@@ -4,107 +4,69 @@ short-title: Supported platforms
-  - /src/get-started/install/windows.md
-  - /src/get-started/install/linux.md
--->
-
-As of the current release,
-Flutter supports deployment of apps on the following platforms 
-as part of Google-tested and best-effort platform tier:
+  * /src/get-started/install/macos.md
+  * /src/get-started/install/windows.md
+  * /src/get-started/install/linux.md
+{% endcomment %}
 
-<div class="table-wrapper" markdown="1">
-|Platform|Version                       |Channels |
-|--------|------------------------------|---------|
-|Android | API 16 (Android 4.1) & above | All     |
-|iOS     | iOS 11 & above               | All     |
-|Linux   | Debian, 64-bit               | All     |
-|macOS   | Mojave (10.14) & above       | All     |
-|Web     | Chrome 84  & above           | All     |
-|Web     | Firefox 72.0 & above         | All     |
-|Web     | Safari on El Capitan & above | All     |
-|Web     | Edge 1.2.0 & above           | All     |
-|Windows | Windows 7 & above, 64-bit    | All     |
-{:.table.table-striped}
-</div>
 
-All channels include master, beta,
-and stable channels.
+## Deploying Flutter
 
-### Google-tested platforms
+As of the current stable release,
+support for deploying Flutter apps is shown in the
+following table:
 
 <div class="table-wrapper" markdown="1">
-|Platform|Version               |
-|--------|----------------------|
-|Android |Android SDK 19–30*    |
-|iOS     |14-15                 |
-|Linux   |Debian 10             |
-|Linux   |Ubuntu 18.04 LTS      |
-|macOS   |Monterey (12) & above |
-|Web     |Chrome 84             |
-|Web     |Firefox 72.0          |
-|Web     |Safari / Catalina     |
-|Web     |Edge 1.2.0            |
-|Windows |Windows 10            |
+|Platform version|<center>Supported</center>|<center>Best effort</center>|<center>Unsupported</center>|
+|----------------|---------|-----------|-----------|
+| Android SDK 21-30        |<center>✓</center>| | |
+| Android SDK 19-20        | |<center>✓</center>| |
+| Android SDK 18 & below   | | |<center>✓</center>|
+| iOS 14-15                |<center>✓</center>| | |
+| iOS 11-13                | |<center>✓</center>| |
+| [iOS 10 & below & `arm7v` 32-bit iOS][] | | |<center>✓</center>|
+| Linux Debian 11          | |<center>✓</center>| |
+| Linux Debian 10          |<center>✓</center>| | |
+| Linux Debian 9 & below   | |<center>✓</center>| |
+| Linux Ubuntu 22.04       |in progress|<center>✓</center>| |
+| Linux Ubuntu 20.04       | |<center>✓</center>| |
+| Linux Ubuntu 18.04 LTS   |<center>✓</center>| | |
+| Linux (any 32-bit platform)| | |<center>✓</center>|
+| macOS Monterey (12) & above|<center>✓</center>| | |
+| macOS Mojave (10.14) - Big Sur (11) | |<center>✓</center>| |
+| macOS High Sierra (10.13) & below | | |<center>✓</center>|
+| web - Chrome 84          |<center>✓</center>| | |
+| web - Firefox 72.0       |<center>✓</center>| | |
+| web - Safari/Catalina    |<center>✓</center>| | |
+| web - Edge 111.0         |<center>✓</center>|           |        |
```

Comment:
Sorry, I think I miscommunicated here. The version of Edge on my machine is 111.0 (that's the very latest version available). We support a much broader range of Edge versions, which should be the same as the range of Chrome versions (e.g. 84 and above), since they share the same engine and version numbering system.